### PR TITLE
elevation points edits

### PR DESCRIPTION
**Pull request by elliotap**
Edited feature attributes in QGIS

- #376 Cabo de Sao Vicente
- #382 Corrected Spelling 
- #439 Expanded Abbreviations
- #438 Renamed Harney Peak

Ran make derived_physical_labels - 10m and 50m files affected

### CODE REVIEW

File: 110m_physical/ne_110m_geography_regions_elevation_points.prj [hunk 1]
```diff
@@ -1 +1 @@
-GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]
\ No newline at end of file
+GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137.0,298.257223563]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]]
```
Comment (nvkelso) [inline 110m_physical/ne_110m_geography_regions_elevation_points.prj:1]: Please remove the PRJ change from this PR.

File: 50m_physical/ne_50m_geography_regions_elevation_points.prj [hunk 2]
```diff
@@ -1 +1 @@
-GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]
\ No newline at end of file
+GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137.0,298.257223563]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]]
```
Comment (nvkelso) [inline 50m_physical/ne_50m_geography_regions_elevation_points.prj:1]: Please remove the PRJ change from this PR.

**Comment by nvkelso** [comment]
Can you also address #382 for `Pico Cristóbal Colón` and #376 `Cabo de Sao Vicente` in this PR to simplify the review since they all touch the same ur file and are simple changes? Thanks!

File: 110m_physical/ne_110m_geography_regions_elevation_points.prj (hunk 1 above)
Comment (elliotap) [reply 110m_physical/ne_110m_geography_regions_elevation_points.prj]: removed change

File: 50m_physical/ne_50m_geography_regions_elevation_points.prj (hunk 2 above)
Comment (elliotap) [reply 50m_physical/ne_50m_geography_regions_elevation_points.prj]: removed change

**Comment by elliotap** [comment]
So far, this has edits for #382 #438 and #439 in the geography_regions_elevation_points set

Where should I look for the make command for the geography_regions_points set? I will add #376 here along with the others.

**Comment by nvkelso** [comment]
The Makefile command is:

> make derived_physical_labels 

Which will generate all the 50m and 110m geography_regions_points and geography_regions_elevation_points theme variants.

That's a pseudo target that groups individual commands like:

> make 50m_physical/ne_50m_geography_regions_points.shp
> make 50m_physical/ne_50m_geography_regions_elevation_points.shp

**Comment by elliotap** [comment]
Thanks! I see what you mean by ur file now. I've updated here with edits to the feature in #376

**Comment by nvkelso** [comment]
Note #376 Cabo de Sao Vicente is in the general ne_10m_geography_regions_points file, while the others are in ne_10m_geography_regions_elevation_points.

**Comment by nvkelso** [comment]
@elliotap These changes look good... there is a slight inconsistency in the `name_*` but that can be remedied with running the Wikidata localization scripts again... which will be done before final release packaging.

**Pull request merged**